<ui version="4.0" >
 <class>NewFileWidget</class>
 <widget class="QWidget" name="NewFileWidget" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>New File</string>
  </property>
  <property name="whatsThis" >
   <string>Select the type of file you wish to create.&lt;br>
&lt;p>
&lt;b>Pic Program&lt;/b>&lt;br>
Creates a new PIC program, with flow chart editor. Select the target device for your program below.
&lt;p>
&lt;b>Circuit&lt;/b>&lt;br>
Creates a new circuit, with drag and drop editor. Real time simulation of the circuit occurs automatically.</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="textLabel1" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>New File Details</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="textLabel1_2_2" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>File Type:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="KListWidget" name="typeIconView" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>500</width>
       <height>150</height>
      </size>
     </property>
     <property name="acceptDrops" >
      <bool>false</bool>
     </property>
     <property name="editTriggers" >
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="dragDropMode" >
      <enum>QAbstractItemView::DragDrop</enum>
     </property>
     <property name="flow" >
      <enum>QListView::LeftToRight</enum>
     </property>
     <property name="resizeMode" >
      <enum>QListView::Adjust</enum>
     </property>
     <property name="viewMode" >
      <enum>QListView::IconMode</enum>
     </property>
     <property name="uniformItemSizes" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QCheckBox" name="addToProjectCheck" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>&amp;Add to project</string>
     </property>
     <property name="shortcut" >
      <string>Alt+A</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QComboBox" name="microSelect" >
     <item>
      <property name="text" >
       <string>Test1</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Test1.5</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Test2</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>KListWidget</class>
   <extends>QListWidget</extends>
   <header>klistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
